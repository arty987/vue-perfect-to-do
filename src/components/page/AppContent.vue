<script>
import AppMain from "@/components/organisms/AppMain.vue";
import AppHeader from "@/components/atoms/AppHeader.vue";

export default {
  name: "AppContent",
  components: {
    AppHeader,
    AppMain
  }
}
</script>

<template>
  <AppHeader/>
  <AppMain/>
</template>