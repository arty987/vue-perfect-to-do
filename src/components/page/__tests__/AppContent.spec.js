import {shallowMount} from '@vue/test-utils'
import AppContent from "@/components/page/AppContent.vue";

describe('AppContent.vue', () => {
    it('Should render AppContent', () => {
        const wrapper = shallowMount(AppContent);
        expect(wrapper.element).toMatchSnapshot();
    });
})