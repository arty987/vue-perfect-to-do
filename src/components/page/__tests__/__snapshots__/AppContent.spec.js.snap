// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContent.vue Should render AppContent 1`] = `
<div
  data-v-app=""
>
  
  <app-header-stub />
  <app-main-stub />
  
</div>
`;
